sample.env

```diff
@@ -16,6 +16,13 @@ SECRET_KEY_BASE=
 BIGBLUEBUTTON_ENDPOINT=
 BIGBLUEBUTTON_SECRET=
 
+# The endpoint and shared secret (first 31 characters of V3 secret) for your Greenlight v3 server.
```

Comment:
This should say `secret_key_base` and we should be taking all of the characters and cutting off all unused chars ourself

Reply:
Was thinking to leverage the size limit to protect against direct usage of the secret  in case of an exposure.

But since we'll be requesting admins to change v3 secret after migrating their resources we can place the key base itself.